### ISSUE TITLE
any chance to update to 1.18.x

### ISSUE THREAD

**Issue by VirtualGaming1** [thebrightspark/Land-Manager#20]
Just wondering if you could update this mod to 1.18.x because I am doing modpack stuff again and I enjoy using this mod to make lots like for shop fronts and houses for my mmo base modpack

**Comment by thebrightspark** [comment]
Yes it's somewhat in the works! I'm updating to 1.16.5 first, then will be straight onto 1.18.2 after 🙂
